automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/UserFixtures.scala

```diff
@@ -89,21 +96,28 @@ trait UserFixtures extends CleanUp with ScaledTimeSpans with Eventually { self:
   private def withSignInReal[T <: AuthenticatedPage](user: Credentials, page: T)
                                                     (testCode: T => Any)
                                                     (implicit webDriver: WebDriver): Unit = {
-    withSignIn(user, {
+
+    logger.info(s"withSignInReal: ${user.email} ...")
+    executeTestCodeWithSignIn(user, {
       new SignInPage(FireCloudConfig.FireCloud.baseUrl).open.signIn(user.email, user.password)
     }, page, testCode)
   }
 
-  private def withSignIn[T <: AuthenticatedPage](user: Credentials, signIn: => Unit,
-                                                         page: T, testCode: T => Any)
-                                                        (implicit webDriver: WebDriver): Unit = {
+  private def executeTestCodeWithSignIn[T <: AuthenticatedPage](user: Credentials, signIn: => Unit,
+                                                                page: T, testCode: T => Any)
+                                                               (implicit webDriver: WebDriver): Unit = {
```

Comment:
I renamed this version of `withSignIn` to `executeTestCodeWithSignIn` to make it easier to distinguish in the logging messages.